src/applications/appeals/996/content/addIssue.jsx

```diff
@@ -1,6 +1,6 @@
 import React from 'react';
 
-import { MAX_LENGTH } from '../constants';
+import { MAX_LENGTH } from '../../shared/constants';
```

Comment:
This `MAX_LENGTH` is using `ISSUE_NAME` so it's not in the shared folder

Reply:
That explains the failing tests!

Reply:
@Mottie So we fixed this one in NOD but the HLR one is the one in shared. This should be okay right?

Reply:
Jes. That would help make it easier to read so it's clear which ones are "shared" since a lot of those are specific to HLR and not NOD.